collections/src/main/scala/strawman/collection/immutable/LazyList.scala

```diff
@@ -475,8 +473,12 @@ object LazyList extends LazyListFactory[LazyList] {
       tlEvaluated = true
       tl
     }
-    def force: Evaluated[A] = Some((head, tail))
-    override def toString: String =
+    def force: LazyList[A] = {
+      head
+      tail.force
```

Comment:
2.12 supports forcing cyclic streams:

```
scala> val s: Stream[Int] = List(1,2,3).toStream #::: s
s: Stream[Int] = Stream(1, ?)

scala> s.force
res6: scala.collection.immutable.Stream[Int] = Stream(1, 2, 3, ...)
```

We should aim for an iterative implementation instead of a (non-tail-) recursive one.

https://github.com/scala/scala/blob/v2.13.0-M3/src/library/scala/collection/immutable/Stream.scala#L254-L276

Reply:
Context: https://github.com/scala/scala/pull/3848. Don't worry about replicating the test cases of that PR here in the strawman repo, the test cases in scala/scala will cover it.

Reply:
Fixed.